model/feature_flags.go

```diff
@@ -7,8 +7,12 @@ type FeatureFlags struct {
 	// Exists only for unit and manual testing.
 	// When set to a value, will be returned by the ping endpoint.
 	TestFeature string
+
+	// Toggle on and off scheduled jobs for cloud user limit emails
+	CloudEmailJobsEnabledMM29999 bool
 }
 
 func (f *FeatureFlags) SetDefaults() {
 	f.TestFeature = "off"
+	f.CloudEmailJobsEnabledMM29999 = false
```

Comment:
In your current tests you are overriding this in the env var?

Reply:
Yes, you can set it via MM_FEATUREFLAGS_<feature flag name> just like other configs